apps/snitch_core/lib/core/data/schema/stock/stock_transfer.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Snitch.Data.Schema.StockTransfer do
+  @moduledoc """
+  Model to track inventory transfer in location
+  """
+  use Snitch.Data.Schema
+  use Snitch.Data.Schema.Stock
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_stock_transfers" do
+    field(:type, :string)
+    field(:reference, :string)
+    field(:number, :string)
+
+    belongs_to(:source_location, StockLocation)
```

Comment:
IMO we can drop the `_location` suffix as there is no ambiguity about their meaning in this context.